CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing
```

Comment:
Can just point to our doc like this: https://github.com/screwdriver-cd/store-cli/blob/master/CONTRIBUTING.md